misc/ballerinalang-data-mapper/src/test/java/org/ballerinalang/datamapper/util/TestUtil.java

```diff
@@ -210,10 +209,7 @@ public static String getResponseString(CompletableFuture completableFuture) {
         if (project.isEmpty()) {
             return diagnostics;
         }
-        for (Module module : project.get().currentPackage().modules()) {
-            diagnostics.addAll(module.getCompilation().diagnostics().diagnostics());
-        }
-
+        diagnostics.addAll(project.get().currentPackage().getCompilation().diagnosticResult().diagnostics());
```

Comment:
Shall we get this to a variable in order to improve the readability?

Reply:
Resolved in 1907cf8